src/VisualStudio/IntegrationTest/TestUtilities/OutOfProcess/Editor_OutOfProc.cs

```diff
@@ -192,6 +192,12 @@ public void DialogSendKeys(string dialogAutomationName, string keys)
             SendKeys(new KeyPress(VirtualKey.K, ShiftState.Ctrl), new KeyPress(VirtualKey.D, ShiftState.Ctrl));
         }
 
+        public void FormatDocumentViaCommand()
```

Comment:
When I tested locally, the test failed saying the key combination was not valid in the current context. I'm not sure if it was a local configuration issue, but the binding isn't part of the intent of this test so I just switched to using the named command.

Reply:
@sharwell Doesn't need to be done as a part of this PR, but it seems we should just only have this; I don't know why we'd ever want to invoke via keys precisely because of the problem you ran into. Profiles sometimes mean this is Ctrl+K Ctrl+D or Ctrl+E Ctrl+D, exactly what we want to avoid.